.vscode/tasks.json

```diff
@@ -8,6 +8,21 @@
     "showReuseMessage": true
   },
   "tasks": [
+    {
+      "label": "Dependencies",
+      "dependsOn": [
+        "CheckVersions",
+        "FoundryUp",
+        "YarnInstall",
+        "Install Protobuf Dependencies",
+        "BuildProtobufs",
+      ],
+      "group": {
+        "kind": "build",
+        "isDefault": true
+      },
+      "dependsOrder": "sequence"
+    },
```

Comment:
is this slow if it's already installed? if so can you just add it to checkVersions? otherwise i'm fine with this.